diff --git ct-ng.in ct-ng.in
index 2fad2243..d08b6ab7 100644
--- ct-ng.in
+++ ct-ng.in
@@ -12,6 +12,8 @@ MAKEFLAGS += --no-print-directory --no-builtin-rules
 
 # This is where ct-ng is:
 export CT_NG:=$(abspath $(lastword $(MAKEFILE_LIST)))
+# This is the directory where ct-ng is:
+export CT_BASE_DIR:=$(dir $(abspath $(firstword $(MAKEFILE_LIST))/..))
 # and this is where we're working in:
 export CT_TOP_DIR:=$(shell pwd)
 
@@ -25,9 +27,9 @@ export CT_LICENSE_DIR
 else
 # Paths and values set by ./configure
 # Don't bother to change it other than with a new ./configure!
-export CT_LIB_DIR:=@pkgdatadir@
-export CT_LIBEXEC_DIR:=@pkglibexecdir@
-export CT_DOC_DIR:=@docdir@
+export CT_LIB_DIR:=$(CT_BASE_DIR)/share/crosstool-ng
+export CT_LIBEXEC_DIR:=$(CT_BASE_DIR)/libexec/crosstool-ng
+export CT_DOC_DIR:=$(CT_BASE_DIR)/share/doc/crosstool-ng
 endif
 
 # This is crosstool-NG version string
diff --git packages/isl/package.desc packages/isl/package.desc
index ffae478d..20f7e024 100644
--- packages/isl/package.desc
+++ packages/isl/package.desc
@@ -1,6 +1,6 @@
 repository='git git://repo.or.cz/isl.git'
 bootstrap='./autogen.sh'
-mirrors='http://isl.gforge.inria.fr'
+mirrors='https://libisl.sourceforge.io'
 relevantpattern='*.*|.'
 milestones='0.12 0.13 0.14 0.15 0.18'
 archive_formats='.tar.xz .tar.bz2 .tar.gz'
diff config/versions/isl.in config/versions/isl.in
--- config/versions/isl.in
+++ config/versions/isl.in
@@ -475,7 +475,7 @@
 
 config ISL_MIRRORS
     string
-    default "http://isl.gforge.inria.fr"
+    default "https://libisl.sourceforge.io"
 
 config ISL_ARCHIVE_FILENAME
     string